docker-compose.yml

```diff
@@ -113,6 +113,7 @@ services:
   # ----------------------------------------------------------------------
   rstudio:
     hostname: rstudio
+    platform: linux/amd64
     image: pecan/base:${PECAN_VERSION:-latest}
```

Comment:
This one raised my eyebrows for a minute because our `rstudio-nginx` image has ARM builds (https://hub.docker.com/r/pecan/rstudio-nginx/tags), but I now see this service doesn't use that image. @robkooper  can you remind me what pecan/rstudio-nginx is used for if not this?